en/docs/deploy/configure-a-read-write-active-directory-user-store.md

```diff
@@ -353,7 +353,7 @@ Possible values:<br />
 SHA - Uses SHA digest method. SHA-1, SHA-256<br />
 MD5 - Uses MD 5 digest method.<br />
 PLAIN_TEXT - Plain text passwords.(Default)</p>
-<p>If you just configure as SHA, It is considered as SHA-1, It is always better to configure algorithm with higher bit value as digest bit size would be increased.<br />
+<p>If you just configure as SHA, it is considered as SHA-1. It is recommended to use SHA-256 since SHA-1 is no longer considered as secure. It is always better to configure an algorithm with a higher bit value since digest bit size would be increased.<br />
```

Comment:
```suggestion
<p>If you just configure as SHA, it is considered as SHA-1. It is recommended to use SHA-256 since SHA-1 is no longer considered as secure. It is always better to configure an algorithm with a higher bit value since the digest bit size will be increased.<br />
```